### PR TITLE
BoF #2 charter issue #5

### PR DESCRIPTION
**Pull request by bledvina**
Attempts to address issue #108

### CODE REVIEW

File: charter/charter-detecting-unwanted-location-trackers.md [hunk 1]
```diff
@@ -47,7 +47,11 @@ The WG is expected to:
 5. Design mechanisms to ensure that devices that do not correctly implement or adhere to the DULT protocol can be detected and excluded from being trackable via crowdsourced location networks.
  * This includes considerations for addressing legacy trackers that cannot update to the DULT protocol.
 
-The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on small and not easily-discoverable accessories. Accessories are considered easily discoverable if they meet one of the following criteria:
```
Comment (ekr) [inline charter/charter-detecting-unwanted-location-trackers.md:50]: I think this "end-to-end" language is a bit confusing. Let me see if I can flesh out the intention here. 

- Standardize the protocols used by (1) the tag to communicate with finding devices (2) the tag to communicate with devices when apparent unwanted tracking is detected (e.g., to cause it to beep).
- Not standardize the protocols use by the finding device to report to the tracking network (e.g., to upload to iCloud), the protocols used to initialize the tag, or the protocols used by the tag owner to query the network for location.

One way to phrase this is that it ought to be enough to allow you to build your own tag that would work with the network, assuming it had the right keying material. 

Does that sound right?

File: charter/charter-detecting-unwanted-location-trackers.md [hunk 2]
```diff
@@ -47,7 +47,11 @@ The WG is expected to:
...
 
-The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on small and not easily-discoverable accessories. Accessories are considered easily discoverable if they meet one of the following criteria: 

+
+* The accessory is larger than 30 cm in at least one dimension. 

+* The accessory is larger than 18 cm x 13 cm in two of its dimensions. 

+* The accessory is larger than 250 cm3 in three-dimensional space.
```
Comment (ekr) [inline charter/charter-detecting-unwanted-location-trackers.md:54]: Now that I see this written down, I'm not sure how helpful it is. I get that we're not worried about someone dropping a bicycle in my backpack, but what's restricting this to small accessories getting us?

**Review by ekr — CHANGES_REQUESTED** [review]

File: charter/charter-detecting-unwanted-location-trackers.md (hunk 2 above)
Comment (tfpauly) [reply charter/charter-detecting-unwanted-location-trackers.md:54]: I think there's a way to say "small things that could be slipped into a bag unnoticed" without having the charter define sizes in centimeters.

File: charter/charter-detecting-unwanted-location-trackers.md [hunk 3]
```diff
@@ -48,7 +48,11 @@ The WG is expected to:
...
 
-The WG will not define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on small and not easily-discoverable accessories. Accessories are considered easily discoverable if they meet one of the following criteria: 

+
+* The accessory is larger than 30 cm in at least one dimension. 

+* The accessory is larger than 18 cm x 13 cm in two of its dimensions. 

+* The accessory is larger than 250 cm3 in three-dimensional space.
```
Comment (tfpauly) [inline charter/charter-detecting-unwanted-location-trackers.md:55]:
```suggestion
The WG will not define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on solving the use case of detecting small and not easily-discoverable accessories, supporting any functionality that is necessary for identifying and recognizing such accessories.
```

**Review by tfpauly — APPROVED** [review]

File: charter/charter-detecting-unwanted-location-trackers.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ The WG is expected to:
 5. Design mechanisms to ensure that devices that do not correctly implement or adhere to the DULT protocol can be detected and excluded from being trackable via crowdsourced location networks.
  * This includes considerations for addressing legacy trackers that cannot update to the DULT protocol.
 
-The WG will not define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on solving the use case of detecting small and not easily-discoverable accessories, supporting any functionality that is necessary for identifying and recognizing such accessories.
```
Comment (tfpauly) [inline charter/charter-detecting-unwanted-location-trackers.md:51]: @ekr does this help explain why the focus would be on the use cases for small devices? Effectively, this lets the protocol work include things like ways to trigger playing a sound or getting a picture of what the accessory looks like, which isn't necessary for the bike case.

Comment (ekr) [reply charter/charter-detecting-unwanted-location-trackers.md:51]: I think it depends on whether the tag is removable, right?

Comment (tfpauly) [reply charter/charter-detecting-unwanted-location-trackers.md:51]: What depends on being removable? I think the main point is to have the focus of the development be on these kinds of devices, and make sure that if there needs to be a way to fetch a URL to get a picture of the device or a command to trigger a sound, those are scoped in.

**Comment by ekr** [comment]
Well, I guess I don't really understand what the relevance of this scoping is. Like, what are the special characteristics of smaller tags that would lead to requirements you wouldn't want to impose on tags in bicycles? 

My bigger point is that suppose you say "tags in big but cheap item X don't need to conform" and then it turns out that the way people tag cheap item X is by making the same transponder and just gluing it to X, so you can take it out and use it for tracking.

**Comment by tfpauly** [comment]
I am thinking of this less as cases where there are requirements we wouldn't want to impose on large objects (bicycles), but rather that if there are features that are _not_ supported by the protocol that are needed for the very small devices (like triggering sounds or fetching a picture) that the work of the WG may not have satisfied the primary use case.

**Comment by ekr** [comment]
OK, I can live with the current text, I guess,

**Pull request merged**